setup.py

```diff
@@ -49,7 +49,7 @@
     install_requires=[
         "addfips>=0.4,<0.5",
         "catalystcoop.dbfread>=3.0,<3.1",
-        "catalystcoop.ferc-xbrl-extractor==0.6.1",
+        "catalystcoop.ferc-xbrl-extractor==0.8.2",
```

Comment:
This was required because 0.6.1 is no longer fetchable on Pypi

Reply:
This seems weird. How is it failing? I'm able to install it with:

```
pip install "catalystcoop.ferc-xbrl-extractor==0.6.1"
pip list | grep xbrl-extractor
catalystcoop.ferc-xbrl-extractor 0.6.1
```
I'm pretty sure that the newer version will not be compatible with the older codebase.

https://pypi.org/project/catalystcoop.ferc-xbrl-extractor/0.6.1/

Reply:
It was failing with `Unable to find specified version, among ["0.8.0", "0.8.1", "0.8.2"]` but now it works...

I've tested this on linux (WSL) and ARMx64 Mac. 

I need your help to package, and deploy it to, both due to lack of knowledge and also some permissions (this is a weird one where it shouldn't be merged into main or dev, but rather create a new branch in this org and tag the release to that branch)

Reply:
(Once you're OK with this PR I'll rebase the two commits, I generally hold off on rebasing until approval)

Reply:
Thank you for finding a fix!

Okay, so we need to keep this change on a little branch by itself, but tag it and use it to do a software release.

This seems like maybe it should be a "post" release since we're just pinning a dependency. Giving it a current date stamp also seems like it might be confusing and suggest that all the changes we've made since last year are included, so I'm`v2022.11.30.post1`  Dealing with the fork/branch/permissions sounds harder than just copying the one-line change over so I will create ` v2022.11.30.post1` branch and add the one-line change.

Reply:
That makes sense to me. For those following along, slack discussion in https://catalystcooperative.slack.com/archives/C01UQPVETGD/p1680798949838099

Please let me know if I can help further